src/views/TradingCompetition/components/BattleCta/index.tsx

```diff
@@ -112,6 +116,10 @@ const BattleCta: React.FC<CompetitionProps> = ({
 
     // User registered and competition finished
     if (hasCompetitionEnded) {
+      // Claim period has ended
+      if (currentPhase.state === OVER) {
```

Comment:
`hasCompetitionEnded` is an amalgamation of the 3 'ended' states - `FINISHED`, `CLAIM` & `OVER`.

https://github.com/pancakeswap/pancake-frontend/blob/2d889f83587b450b278f6343ca5cfd67e9bcc771/src/views/TradingCompetition/index.tsx#L102

Tricky because there are some ui changes consistent across all three, hence my creation of that condition, but then there are also some changes specific to each.